src/util/std_expr.h

```diff
@@ -4121,6 +4121,9 @@ class false_exprt:public constant_exprt
   }
 };
 
+/// \brief returns true_exprt if given true and false_exprt otherwise
+constant_exprt make_boolean_expr(bool);
```

Comment:
Two nit picks:
1) `\brief` is redundant and using a full sentence (starting with an uppercase latter and ending in a full stop) would be nice.
2) I'm wondering whether `std_expr.h` is the right place for this. It's already a way-too-big file, and this doesn't fit particularly well I think. (With the additional thought that `conjunction`, `disjunction` should move as well.)

Reply:
2. expr_util.h?

Reply:
I do think that would be a better place, yes.